sormas-app/app/src/main/java/de/symeda/sormas/app/visit/edit/VisitEditFragment.java

```diff
@@ -86,6 +87,13 @@ public void onLayoutBinding(FragmentVisitEditLayoutBinding contentBinding) {
 		VisitValidator.initializeVisitValidation(contact, contentBinding);
 
 		contentBinding.setVisitStatusClass(VisitStatus.class);
+
+		boolean isExternalVisit = record.getOrigin().equals(VisitOrigin.EXTERNAL_JOURNAL);
+		if (isExternalVisit) {
+			contentBinding.visitVisitDateTime.setEnabled(false);
```

Comment:
Do we have a definition of which fields should be editable/readonly for external visits?

Reply:
If the external visit should not be editable at all, then the edit activity should not open, but the read activity instead.